src/chrome-finder.ts

```diff
@@ -197,12 +197,22 @@ export function win32() {
     installations.push(customChromePath);
   }
 
+  // Search common locations for chrome.exe
   prefixes.forEach(prefix => suffixes.forEach(suffix => {
     const chromePath = path.join(prefix, suffix);
     if (canAccess(chromePath)) {
       installations.push(chromePath);
     }
   }));
+
+  // Search %PATH% for chrome.exe
+  process.env.Path?.split(';')?.forEach(prefix => {
```

Comment:
- shouldn't this be `PATH`? Did it work for you as `Path`? Or are env vars in windows cased like that?
- Seems we could search PATH in mac (darwin in our code) and linux too
- I think the prefixes array about could have this added (`...process.env.Path.split(';')`), and this extra loop can be removed

Reply:
- I got confused between `Path` and `PATH` because of https://github.com/nodejs/node/issues/20605.
- I'll take a look at adding that too.
- Replaced the second loop.

Reply:
On Linux, `which` is used, so manual `PATH` parsing is unneeded.

On Darwin, `lsregister` is used, which _I think_ means it looks up apps:
> The Launch Service database controls the Finder's 'Open With' right click, contextual menu.

Reply:
oh, found a reason to use `where` over doing it manually: in windows there's a way to use semicolons for individual items in the PATH... https://superuser.com/questions/584870/how-can-i-add-a-folder-containing-a-semicolon-to-the-windows-path . so if we use `where` we don't have to worry about the particulars of parsing PATH at all, which apparently isn't as trivial as unix (in unix/POSIX it is _impossible_ to escape the semicolon)